ddtrace/tracer/option.go

```diff
@@ -30,6 +30,12 @@ type config struct {
 	// serviceName specifies the name of this application.
 	serviceName string
 
+	// envService specifies the service tag set with DD_SERVICE that will be sent with all traces unless overridden.
+	envService string
```

Comment:
This is necessary to keep track of the environment service name, so we can set the version `if span.Service == envService`